assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6

```diff
@@ -251,19 +251,17 @@ export default createWidget("discourse-reactions-actions", {
               this.setCurrentUserReaction();
               later(() => {
```

Comment:
do we still need this later ?

Reply:
Yes, later is still needed here

Reply:
can you elaborate why ?

Reply:
also if it's needed please look at what I do with later in others parts of the code, we can let a later uncontrolled like this

Reply:
If we don't use it later here, the `drop animation` stops working, This is due to the `this.scheduleRerender()` it rerenders the page which eventually stops the animation.

Reply:
yes understood, but still looks at what I do with later/cancel please

Reply:
Used cancel in the latest commit😅 do have a look